app/controllers/v1/users/mutual_friends_controller.rb

```diff
@@ -3,13 +3,18 @@
 class V1::Users::MutualFriendsController < ApplicationController
   before_action :pundit_user
   before_action do
-    @page = set_page
+    @page = set_page.to_i
   end
 
   def index
```

Comment:
Too much logic too, maybe something to abstract in a class?

Reply:
Would you suggest using a PORO just like I did with the `services/json_web_token.rb` file?

Reply:
Did the same fix here as with friends_controller per your suggestion there :) Thanks!